### ISSUE TITLE
Show i18n best practice in example

### ISSUE THREAD

**Issue by msporny** [w3c/vc-data-integrity#211]
A quick i18n review by @aphillips requested that we demonstrate i18n best practices:

https://github.com/w3c/i18n-activity/issues/1771

This issue tracks that concern such that it's addressed before the DI specs go to CR.

**Comment by msporny** [comment]
We should add an example that demonstrates how to secure an internationalized string.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-10-17-vcwg-special#section1-1) on 2023-10-17

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.1. Show i18n best practice in example (issue vc-data-integrity#211)

_See github issue [vc-data-integrity#211](https://github.com/w3c/vc-data-integrity/issues/211)._




**Manu Sporny:** We have started the CR transition for the DI specs and JSON schema, I made a mistake and didn't properly request i18n review, but fortunately Addison took care of this for us. He would like to see an example of an i18n string.  
… We have a hello world example that I'd like to keep as this is helpful in all specs generally, but I want to add an i18n example and that needs to go in before CR.  
… I make a PR to do that and use the mechanism that we all agreed to for that.  
… I would like to merge it before 7 days, it's informative, no normative text.  

**Brent Zundel:** The plan, as I understand it, is to raise a PR to add a non-normative i18n example to data integrity and then merge it at some point before the 7 day window.  
… Any objections to that plan of action?  
… No objections here, thank you, you're good to go.  

</details>

**Comment by msporny** [comment]
PR #213 has been raised to address this issue. This issue will be closed once PR #213 has been merged.

**Comment by aphillips** [comment]
Pasting in the I18N comment:

---
Example 1
https://www.w3.org/TR/vc-data-integrity/#example-a-simple-json-data-document

>```
>{
>  "title": "Hello world!"
> };
>```

Also found in the other docs in this review, e.g.

- https://www.w3.org/TR/vc-di-eddsa/#example-an-ed25519-digital-signature-expressed-as-a-ed25519signature2020
- https://www.w3.org/TR/vc-di-eddsa/#example-credential-without-proof 
- https://www.w3.org/TR/vc-di-eddsa/#example-credential-without-proof-1
- https://www.w3.org/TR/vc-di-ecdsa/#example-an-ecdsa-p-256-digital-signature-expressed-as-a-dataintegrityproof

There are a number of examples that have title as a string (as shown above). These ought to use the language and direction metadata we've discussed in data model or use a non-language string as the example.

**Comment by msporny** [comment]
> There are a number of examples that have title as a string (as shown above).

The examples in the core spec have been updated to use a non-language string as the example. The test vectors will need to be updated later as @Wind4Greg is soon-to-be/currently on leave and unable to update the test vectors. Since the test vectors are non-normative, we can update them at any time. I've added a tracking issue for this to make sure that we do this as soon as @Wind4Greg is back: https://github.com/w3c/vc-data-integrity/issues/218
